_posts/2020-06-02-tapping-glass.markdown

```diff
@@ -74,7 +74,7 @@ it's also a more complicated shape, would need rigging, and demand soft body phy
 
 Physics simulations tend to be slow, and they would generally be unviable for speed art.
 However, they're also free animation that you don't need to make by hand.
-Breaking glass is a classic interesting visual, and it would mean I only need the small amount of animation to set up the breaking glass, and the rest comes for free.
+Breaking glass is a classic and interesting visual, and it would mean I only need the small amount of animation to set up the breaking glass, and the rest comes for free.
```

Comment:
```suggestion
Breaking glass is a classic interesting visual, and it would mean I only need the small amount of animation to set up the breaking glass, and the rest comes for free.
```

Reply:
Alright, the commit was an honest mistake, it's fine, I'll just rollback to before it.